src/providers/events.js

```diff
@@ -96,7 +106,46 @@ export function createEventsObject(openmct, parentKey, namespace) {
         }
     };
 
-    return eventsObject;
+    return baseEventObject;
+}
+
+export function createEventSeverityObjects(openmct, parentEventObject, namespace) {
+    const childSeverityObjects = [];
+    for (const severity of SEVERITY_LEVELS) {
+        const severityIdentifier = {
+            key: `${parentEventObject.identifier.key}.${severity}`,
+            namespace
+        };
+
+        const severityName = `${parentEventObject.name}: ${severity}`;
+
+        const severityEventObject = createEventObject(
+            openmct,
+            parentEventObject.identifier.key,
+            namespace,
+            severityIdentifier,
+            severityName,
+            OBJECT_TYPES.EVENT_SPECIFIC_SEVERITY_OBJECT_TYPE
+        );
+
+        childSeverityObjects.push(severityEventObject);
+    }
+
+    return childSeverityObjects;
+}
+
+export async function getEventSources(url, instance) {
+    const eventSourceURL = `${url}/api/archive/${instance}/events/sources`;
```

Comment:
I think this should be:
const eventSourceURL = `${url}api/archive/${instance}/events/sources`;

following the pattern of all the other constructed URLs that don't have a / after the ${url}